librz/include/rz_core.h

```diff
@@ -857,6 +805,55 @@ RZ_API void rz_core_analysis_name_free(RZ_NULLABLE RzCoreAnalysisName *p);
 RZ_API RZ_OWN RzCoreAnalysisName *rz_core_analysis_name(RZ_NONNULL RzCore *core, ut64 addr);
 RZ_API bool rz_core_analysis_rename(RZ_NONNULL RzCore *core, RZ_NONNULL const char *name, ut64 addr);
 
+/* cgraph.c */
+/**
+ * \brief RzGraph format for print/convert
+ */
+typedef enum {
+	RZ_CORE_GRAPH_FORMAT_VISUAL = 0,
+	RZ_CORE_GRAPH_FORMAT_TINY,
+	RZ_CORE_GRAPH_FORMAT_SDB,
+	RZ_CORE_GRAPH_FORMAT_GML,
+	RZ_CORE_GRAPH_FORMAT_DOT,
+	RZ_CORE_GRAPH_FORMAT_JSON,
+	RZ_CORE_GRAPH_FORMAT_JSON_DISASM,
+	RZ_CORE_GRAPH_FORMAT_CMD,
+	RZ_CORE_GRAPH_FORMAT_ASCII_ART,
+	RZ_CORE_GRAPH_FORMAT_UNK,
+} RzCoreGraphFormat;
+
+/**
+ * \brief RzGraph type, see ag?
```

Comment:
Referring to the `ag?` here is a bad idea - we could change the command, and it will be completely out of sync. I recommend to put full description here, after all Rizin is also a library, not just a tool